### PR TITLE
MWPW-141219 FaaS Report

### PR DESCRIPTION
**Pull request by Brandon32**
* Creating a report of unique FaaS authoring variations.
* Add Excel Reporter object logging

Resolves: [MWPW-141219](https://jira.corp.adobe.com/browse/MWPW-141219)

### CODE REVIEW

**Review by meganthecoder — APPROVED** [review]

File: test/bulk-update/bulk-update.test.js [hunk 1]
```diff
@@ -65,6 +85,12 @@ describe('BulkUpdater', () => {
       expect(data).to.deep.equal(expectedData);
     });
 
+    it('should return an empty array if source is not provided', async () => {
```
Comment (JasonHowellSlavin) [inline test/bulk-update/bulk-update.test.js:88]: nit: `should return` -> `returns`

File: test/faas-variations/faas-variations-report.test.js [hunk 2]
```diff
@@ -0,0 +1,166 @@
...
+
+    tests.forEach(([input, expectedOutput]) => {
+      it(`converts correct block information from '${input}' to '${expectedOutput.variant}'`, () => {
+        expect(getBlockInfo(input)).to.deep.equal(expectedOutput);
+      });
+    });
+  });
+
+  describe('getVariantName', () => {
+    const tests = [[0, 'A'], [25, 'Z'], [26, 'AA'], [51, 'AZ'], [52, 'BA'], [77, 'BZ'], [78, 'CA'], [701, 'ZZ'], [702, 'AAA']];
```
Comment (JasonHowellSlavin) [inline test/faas-variations/faas-variations-report.test.js:24]: might be good to put something unexpected in here to test mixed inputs?

**Review by JasonHowellSlavin — APPROVED** [review]
looks great, small nits.

File: test/bulk-update/bulk-update.test.js (hunk 1 above)
Comment (Brandon32) [reply test/bulk-update/bulk-update.test.js]: 👍

File: test/faas-variations/faas-variations-report.test.js (hunk 2 above)
Comment (Brandon32) [reply test/faas-variations/faas-variations-report.test.js:24]: Added a test for unexpected inputs

**Pull request merged**